privacyidea/models.py

```diff
@@ -725,10 +728,12 @@ class Config(TimestampMethodsMixin, db.Model):
 
     @log_with(log)
     def __init__(self, Key, Value, Type=u'', Description=u''):
-        self.Key = unicode(Key)
-        self.Value = unicode(Value)
-        self.Type = unicode(Type)
-        self.Description = unicode(Description)
```

Comment:
I removed the conversions of ``Key``, ``Type`` and ``Descriptions`` to unicode here: My reasoning was that they should always be strings anyway. However, then I realized they can be ASCII bytestrings, and converting them to ``unicode`` could help us get rid of the SQLAlchemy warnings like
```
SAWarning: Unicode type received non-unicode bind param value '...'.
```
So I'm not really sure what to do here :)